frontend/app/settings/page.tsx

```
@@ -28,51 +35,222 @@ const DetailsSection = () => {
+                  resolved.
+                </span>
+              )}
+            </DialogDescription>
+          </DialogHeader>
+
+          {/* Show error message if mutation failed */}
+          {leaveWorkspaceMutation.error && (
+            <div className="mt-4 rounded-md bg-red-50 p-3">
+              <p className="text-sm text-red-700">{leaveWorkspaceMutation.error.message}</p>
+            </div>
           )}
```

ESLint

[error] Potential leaked value that might cause unintentionally rendered values or rendering crashes

(react/jsx-no-leaked-render)